docs/add-um-users.html

```diff
@@ -3,5 +3,52 @@
 title: Add U-M Users
 ---
 
-<h1 class="mt-5">Add U-M Users Documentation</h1>
-<p>Follow documentation for this Feature</p>
+<h1 class="mt-5">Add U-M Users</h1>
+<p>Bulk adding U-M users to course sections with Canvas Course Manager is an administrator feature that offers some benefits over the People tool method:</p>
+<ul>
+  <li>You can add users from a CSV file.</li>
+  <li>You can create a section on the fly and add users to it.</li>
+  <li>You can add users to multiple preexisting sections at one time.</li>
+  <li>Prevent Canvas course invitation messages and email notifications from being sent to new course members.</li>
+</ul>
+<h2 class="h5">Who Can Add U-M users</h2>
+<p>Bulk adding U-M users to sections is limited to Canvas administrator roles:</p>
+<ul>
+  <li>Root Account Administrator</li>
+  <li>Sub-Account Administrator</li>
+  <li>Support Consultant</li>
+  <li>Teachers and other course managers can request assistance by contacting <a href="mailto:4help@umich.edu">4help@umich.edu</a>.</li>
```

Comment:
I still think this shouldn't be a `li`, instead, just a `p` outside the list, but @melindakraft can decide.

Reply:
I will keep it what I have since @melindakraft prefers that way.